peg/Parser.hx

```diff
@@ -207,17 +211,30 @@ class Parser {
 		return name;
 	}
 
+	function mapParsedType(type:String):PType {
+		return switch(type) {
```

Comment:
instead of listing all possible casing in cases it's better to switch on `type.toLowerCase()`

Reply:
done in 09f1f4c